Dockerfile.ci

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.17 as builder
+
+ARG ARCH=linux64
+
+# Set Environment Variables
+SHELL ["/bin/bash", "-c"]
+ENV HOME /app
+ENV CGO_ENABLED 0
+
+WORKDIR /app
+
+# Build Application
+COPY . .
+RUN make deps
+RUN NO_DIRTY=true make build
+RUN chmod +x /app/build/infracost
+
+# Application
+FROM alpine:3.15 as app
+# Tools needed for running diffs in CI integrations
+RUN apk --no-cache add bash git jq nodejs npm
+
+# Install the latest compost version
+RUN npm install -g @infracost/compost
+
+WORKDIR /root/
+
+# Scripts are used by CI integrations and other use-cases
+COPY scripts/infracost-comment.sh /usr/bin/infracost-comment
+COPY --from=builder /app/build/infracost /usr/bin/
+
+ENV INFRACOST_SKIP_UPDATE_CHECK='true'
```

Comment:
do we not want CI logs to show upgrades when they're available? what do other tools like Pulumi do?

Reply:
In existing CI integrations we set this. If the user is using the `ci-0.9` tag then it will update automatically. So this will only make a difference if they are locking to a patch version (e.g. `ci-0.9.16`). In this case I don't think we should show the message since the user is explicitly setting that version. I don't think it really matters either way since the message will also not be obvious since it will only appear in the CI logs.

Reply:
Good point, I had forgotten about `ci-0.9` :)